src/pal/src/configure.cmake

```diff
@@ -366,6 +366,9 @@ check_cxx_source_runs("
 #include <stdlib.h>
 #include <time.h>
 #include <sys/time.h>
+#ifdef (__linux__)
```

Comment:
@janvorli , Thank you for your comment. The "__LINUX__"  is not valid. I will  remove parentheses because I have just used for safety.